### ISSUE TITLE
ONNX support

### ISSUE THREAD

**Issue by arition** [freyjaSubOCR/freyja-sub-ocr-electron#2]
The Open Neural Network Exchange (ONNX) is an open standard for representing machine learning models. The biggest advantage of ONNX is that it allows interoperability across different open source AI frameworks, which itself offers more flexibility for AI frameworks adoption.

ONNX provides TensorRT (Windows / LInux, fast Nvidia GPU backend), DirectML (Windows, can be used on all GPUs) and WebGL (all platforms, can be used on all GPUs) backends, which have big advantages compared to pyTorch backend.

Currently, the official node-js backend does not support any GPU functionality. Considering running a python backend to do all of the ML stuffs?

https://www.npmjs.com/package/onnxruntime (node-js CPU only)

https://github.com/microsoft/onnxjs (WebGL, not all ops we need are implemented)
